cli/handleSysCheck.js

```diff
@@ -13,13 +23,115 @@ const renderHelp = () => `
     $ yeep syscheck --config=yeep.config.js
 `;
 
+const validateMongo = async (uri, spinner) => {
+  const client = await MongoClient.connect(
+    uri,
+    { useNewUrlParser: true }
+  );
+
+  spinner.stopAndPersist({
+    symbol: chalk.green('✔'),
+    text: 'Connected to database',
+  });
+
+  const db = client.db();
+  try {
+    // TODO: Verify that the uri has the username:password format if the authentication fails!
+    const buildInfo = await db.admin().buildInfo();
+    const isVersionValid = buildInfo.versionArray[0] >= 4;
+    if (!isVersionValid) {
+      spinner.fail(`Mongo version ${buildInfo.version} needs to be higher than 4.0.0`);
```

Comment:
How about we slightly change the error message to something more descriptive, i.e. `Incompatible MongoDB version detected; yeep requires v.4+, current version  is ${buildInfo.version}`